src/fu-util.c

```diff
@@ -463,6 +463,179 @@ fu_util_perhaps_show_unreported (FuUtilPrivate *priv, GError **error)
 	return fu_util_report_history (priv, NULL, error);
 }
 
+static GString *
+fu_util_get_remote_warning (FuUtilPrivate *priv, FwupdRemote *remote, GError **error)
+{
+	GString *str = g_string_new (NULL);
+
+	/* this is designed as a fallback; the actual warning should ideally
+	 * come from the LVFS instance that is serving the remote */
+	g_string_append_printf (str, "%s\n",
+				/* TRANSLATORS: show the user a warning */
+				"Your distributor may not have verified any of "
```

Comment:
If you're keeping the fallback in place, may as well do the OS string replacement here too in this text.

Reply:
we get `desc_plain_str` from `fu_util_get_component_agreement` and `fu_util_get_remote_warning` -- and they both get the replacements done unless I'm missing something? Or you mean put the `$OS_RELEASE:NAME$` in the generic warning? If the latter I wasn't sure 100% if that made sense as we might actually be enabling something actually provided by the distro.

Reply:
I meant putting $OS_RELEASE:NAME$ in the generic warning, but yeah you're right you have no idea the provider of the remote in the generic case. 

That's part of why i'm not sure if there should be a warning in the generic case at all.

Reply:
Maybe we just make the generic warning a bit more tame, e.g.

    Your distributor may not have verified any of the firmware updates for compatibility with your system or connected devices.
    Enabling this remote is done at your own risk.

Reply:
That's probably fine if you're going to have something.

Reply:
Fixed, thanks.